galaxy_ng/app/api/v3/serializers/task.py

```diff
@@ -1,15 +1,56 @@
 import logging
-
+from typing import Any, Dict, Optional
 from rest_framework import serializers
+from drf_spectacular.utils import extend_schema_field
+from pulpcore.app.serializers import ProgressReportSerializer
+from pulpcore.plugin.models import Task
+from galaxy_ng.app.models import CollectionSyncTask
 
-from galaxy_ng.app.api.ui.serializers.base import Serializer
 
 log = logging.getLogger(__name__)
 
 
-class TaskSerializer(Serializer):
+class TaskSerializer(serializers.ModelSerializer):
     pulp_id = serializers.UUIDField(source='pk')
     name = serializers.CharField()
     state = serializers.CharField()
     started_at = serializers.DateTimeField()
     finished_at = serializers.DateTimeField()
+    created_at = serializers.DateTimeField(source='pulp_created')
```

Comment:
@rochacbruno I would recommend splitting these new fields out into a `TaskDetailSerializer` that is used for the `tasks/:id/` endpoint and leaving the list endpoint as a reduced set of information. This is consistent with how some of our other endpoints work such as collections (https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/api/ui/serializers/collection.py#L96) where some information is hidden from the list view to prevent sending too much data.

Reply:
I'm less of a fan of that pattern, especially for /v3/ endpoints. I kind of like it when the v3 endpoints
ViewSets are moreorless ModelViewSets with some tweaks.

For /ui/ cases, I don't have a strong opinion.  Though I might lean towards sep endpoints for `/ui/v1/task-detail` and `/ui/v1/task-summary`, each supporting list and retrieve.  But that isn't a pattern we use at the moment, so doesn't matter. 

But I would like to try to keep the viewsets with multiple large/small serializers out of /v3/ if we can.
(More than likely, we will end up doing something similar once ansible-galaxy traffic picks up, but alas)